src/options.ts

```diff
@@ -81,6 +81,19 @@ export const getSourcemaps = (): string[] => {
   return sourcemapsOption.split(' ');
 };
 
+/**
+ * Dist is optional, but should be a string when provided.
+ * @returns string
+ */
+export const getDist = (): string | undefined => {
+  const distOption: string = core.getInput('dist');
```

Comment:
`core.getInput('dist')` gets that env var `$INPUT_DIST` that GH creates on behalf of each input.